### DISCUSSION TITLE
Gitea git commit status

### DISCUSSION THREAD

**Discussion by DarthHTTP** [fluxcd/flux2#979] [Proposals]
Hi,

Git commit status for Github & others is a very nice functionality thanks for that, am wondering if there is any work in progress for Gitea which supports this via API call. 

I didn't search documentation deep enough but i found a real case scenario which outlines the use of the API: https://github.com/go-gitea/gitea/issues/14059

What component of flux2 does this ? Was browsing code for notification-controller but couldn't find anything.

TIA

**Comment by stefanprodan** [comment]
Here is the GitHub code: https://github.com/fluxcd/notification-controller/blob/main/internal/notifier/github.go in the same package you can find GitLab/BitBucket/Azure DevOps commit status updates.
